app/views/travel_requests/_form.html.erb

```diff
@@ -56,7 +56,7 @@
       <input-text label="Purpose" id="travel_request_purpose" width="expand"
         name="travel_request[purpose]" value="<%= request_change_set.purpose %>"
         helper="Specify the reason this trip is necessary for your job"
-        :options="<%= request_change_set.travel_category_options %>" required></input-text>
```

Comment:
This is just a cleanup item: the InputText vue component doesn't have an options attribute, so this was being ignored anyway.